src/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -501,6 +501,7 @@ private void CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtoc
             if (message.Failed)
             {
                 StartSendAuthResetSignal(null, asyncRequest, ExceptionDispatchInfo.Capture(new AuthenticationException(SR.net_auth_SSPI, message.GetException())));
+                //throw new AuthenticationException(SR.net_auth_SSPI, message.GetException());
```

Comment:
why is this commented out?

Reply:
leftover. but I did not expect people to comment on draft PR. I use this to do test across CI OSes. This is not quite ready for review.